peer/brontide.go

```diff
@@ -2505,11 +2491,18 @@ func (p *Brontide) handleLocalCloseReq(req *htlcswitch.ChanClose) {
 			return
 		}
 
+		// Optimistically try a link shutdown, erroring out if it
+		// failed.
+		if err := p.tryLinkShutdown(chanID); err != nil {
+			peerLog.Errorf(err.Error())
```

Comment:
I think the log could be more verbose, something like the link failed to shut down for easier debugging.

Reply:
made more verbose